pkg/scheduler/apis/config/v1/defaults.go

```diff
@@ -102,6 +103,7 @@ func setDefaults_KubeSchedulerProfile(prof *configv1.KubeSchedulerProfile) {
 
 // SetDefaults_KubeSchedulerConfiguration sets additional defaults
 func SetDefaults_KubeSchedulerConfiguration(obj *configv1.KubeSchedulerConfiguration) {
+	logger := klog.TODO() // called by generated code that doesn't pass a logger
```

Comment:
moving this discussion here: https://github.com/kubernetes/kubernetes/pull/111155/files#r1096010921

Reply:
Can you open an issue and reference it here?

Reply:
https://github.com/kubernetes/kubernetes/issues/115724

Reply:
Reference the issue in the code comment